client/my-sites/sidebar/sidebar.jsx

```diff
@@ -165,8 +169,10 @@ export class MySitesSidebar extends Component {
 				link={ isPreviewable ? '/view' + this.props.siteSuffix : siteUrl }
 				onNavigate={ this.onNavigate }
 				icon="computer"
-				preloadSectionName="preview"
-			/>
+				preloadSectionName="preview">{
+					externalButton
```

Comment:
This formatting looks a little weird to me. Haven't seen it anywhere but I may be wrong. In this place, I would just put `{ externalButton }` together on one line.